src/ray/protobuf/gcs.proto

```diff
@@ -267,6 +267,9 @@ message ResourceDemand {
   // The number of requests for which there is no node that is a superset of
   // the requested resource shape.
   uint64 num_infeasible_requests_queued = 3;
+  // The number of requests of this shape still queued in CoreWorkers that this
+  // raylet knows about.
+  int64 backlog_size = 4;
```

Comment:
Open question: should we also include some kind of `total_backlog_size` in `HeartbeatBatchTableData`?